src/main/java/seedu/guest/logic/commands/EditCommand.java

```diff
@@ -40,6 +42,7 @@ public class EditCommand extends Command {
             + "[" + PREFIX_NAME + "NAME] "
             + "[" + PREFIX_PHONE + "PHONE] "
             + "[" + PREFIX_EMAIL + "EMAIL] "
+            + "[" + PREFIX_NUMBER_OF_GUESTS + "NUMBER OF GUESTS] "
```

Comment:
Should this be NUMBER_OF_GUESTS?

Reply:
Yes it should. Changed made in the recent commit.